### PR TITLE
Fix tenant menu in collapsed sidebar

### LINKED ISSUE #5928: Tenant menu overflows collapsed sidebar
### Package

filament/filament

### Package Version

v3

### Laravel Version

10

### Livewire Version

v2

### PHP Version

PHP 8.2

### Problem description

When tenancy is enabled and the sidebar is collapsible, the tenant menu stays inside the confines of the collapsed sidebare and extends over the edge of the page:

image

### Expected behavior

The tenant menu opens normally in full width even with collapsed sidebare

### Steps to reproduce

Configure a context with tenancy and collapsible sidebar, add two tenants for a user, sign in as the user and collapse the sidebar, then click the tenant initials to open the menu

### Reproduction repository

https://github.com/josefbehr/filament-v3-tenant-menu-collapsible-demo

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by zepfietje**

### CODE REVIEW

File: packages/app/resources/views/components/layouts/app/sidebar/index.blade.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
         x-bind:class="$store.sidebar.isOpen ? 'filament-sidebar-open translate-x-0' : '-translate-x-full rtl:translate-x-full'"
     @endif
     @class([
-        'filament-sidebar fixed inset-y-0 start-0 z-20 flex h-screen w-[--sidebar-width] flex-col overflow-hidden bg-white transition-all dark:bg-gray-800 lg:bg-transparent lg:dark:bg-transparent lg:z-0',
```
Comment (zepfietje) [inline packages/app/resources/views/components/layouts/app/sidebar/index.blade.php:19]: Not too sure if this breaks other cases, but let's see.

File: packages/app/resources/views/components/tenant-menu.blade.php [hunk 2]
```diff
@@ -21,7 +21,10 @@
 
 {{ filament()->renderHook('tenant-menu.before') }}
 
-<x-filament::dropdown placement="bottom-start">
+<x-filament::dropdown
+    placement="bottom-start"
+    teleport
```
Comment (danharrin) [inline packages/app/resources/views/components/tenant-menu.blade.php:26]: Does the user menu also need teleporting? Also we should probably add a `filament-tenant-menu` class for consistency

Comment (zepfietje) [reply packages/app/resources/views/components/tenant-menu.blade.php:26]: User menu doesn't need teleporting as it's not inside the sidebar.

Comment (zepfietje) [inline packages/app/resources/views/components/tenant-menu.blade.php:26]:
```suggestion
    teleport
    class="filament-tenant-menu"
```

**Pull request merged**